src/Application/PSPStubber.php

```diff
@@ -5,6 +5,7 @@
 use Ramsey\Uuid\Uuid;
 
 /**
+ * @deprecated - Will put in a better solution for MAT-327
```

Comment:
Marking this as deprecated so that any usages get flagged by Psalm. Have added the usages here to the baseline so the PR can be merged but then I can do another PR to remove other usages of this PSPStubber class until it can be deleted.